.github/workflows/linters.yml

```diff
@@ -0,0 +1,20 @@
+name: Linters
+
+on: pull_request
+
+jobs:
+  rubocop:
+    name: Rubocop
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: ">=3.1.x"
+      - name: Setup Rubocop
+        run: |
+          gem install --no-document rubocop -v '>= 1.0, < 2.0' # https://docs.rubocop.org/en/stable/installation/
+          [ -f .rubocop.yml ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/ruby/.rubocop.yml
+      - name: Rubocop Report
+        run: rubocop --color
```

Comment:
- [ ] It would be nice if you use the proper linter setup because, in this linter setup, you don't have Stylelint linter which is crucial for your project because your styling files errors can cause breaking the whole application styling :fearful: So stylint should be there to check every error :+1: to prevent your application's styling from breaking :+1:

**Question**
Now you can say that for now, I don't have any styling so it will throw an error and it doesn't make any sense that this pull request should have stylelint linter because for now, this project doesn't have any styling.

**Answer**
First things first, you always set up your application and linters at first without doing any code in that project so you can focus on the code in the next milestone without losing any focus from your original work :+1: and some times we forgot the include linters later then it causes an issue because then we see a lot of errors :thinking:
Secondly, if it will throw an error then you can some basic styling in that file so your check that basic styling and pass :+1: 

**Basic Styling**

```
* {
  padding: 0;
  margin: 0;
  box-sizing: border-box;
}
```